docs/use-cases/tokenization/index.page.tsx

```diff
@@ -0,0 +1,395 @@
+    title: "Trustlines & Authorized Trustlines",
+    description: (
+      <>
+        <div className="bold">{translate("Trustlines:")}</div>
+        <div className="card-text">
+          {translate("No spamming of wallets without permission.")}
+        </div>
+        <div className="bold">{translate("Authorized Trustlines:")}</div>
+        <div className="card-text">
+          {translate(
+            "Enable specific accounts to hold issued tokens by setting up allowlisting."
+          )}
+        </div>
+      </>
+    ),
+  },
+  {
+    id: "freeze-clawbacks",
+    title: "Freeze & Clawbacks",
+    description: (
+      <>
+        <div className="bold">{translate("Freeze:")}</div>
+        <div className="card-text">
+          {translate(
+            "If you see signs of suspicious activity, you can suspend trading of your token while investigating the issue."
+          )}
+        </div>
+        <div className="bold">{translate("Clawback:")}</div>
+        <div className="card-text">
+          {translate(
+            "Recover tokens distributed to accounts in error: for example, reclaim funds sent to an account sanctioned for illegal activity."
+          )}
+        </div>
+      </>
+    ),
+  },
+  {
+    id: "ntf-tokens",
+    title: "Non-transferable Tokens",
+    description: (
+      <>
+        {translate(
+          "Native support for nontransferable items such as identity tokens, airline credits, and consumer rewards, honored by all on-chain participants."
+        )}
+      </>
+    ),
+  },
+  {
+    id: "royalties",
+    title: "Royalties",
+    subtitle: "Secures Creator Compensation:",
+    description: (
+      <>
+        {translate(
+          "Reliably collect your percentage of the sale price of your tokens."
+        )}
+      </>
+    ),
+  },
+];
+
+const stats = [
+  {
+    id: "nfts-minted",
+    stat: "20K",
+    description: "NFTs minted",
+  },
+  {
+    id: "nft-trading-vol",
+    stat: "~1000",
+    description: "NFT trading volume",
+  },
+  {
+    id: "network-speed",
+    stat: "0.4",
+    description: "Network speed (sec)",
+  },
+  {
+    id: "fee-per-tx",
+    stat: "$0.001",
+    description: "Avg fee per NFT tx",
+  },
+];
+
+const projects = [
+  {
+    id: "xrpcafe",
+    label: "XRP cafe",
+    url: "https://xrp.cafe",
+  },
+  {
+    id: "onXRP",
+    label: "onXRP",
+    url: "https://nft.onxrp.com",
+  },
+  {
+    id: "xaman",
+    label: "Xaman labs",
+    url: "https://github.com/XRPL-Labs/Xaman-App",
+  },
+  {
+    id: "amy",
+    label: "amy.network",
+    url: "https://token.amy.network",
+  },
+  {
+    id: "sologenic",
+    label: "Sologenic",
+    url: "https://sologenic.org/trade",
+  },
+  {
+    id: "carbonland",
+    label: "Carbonland trust",
+    url: "https://www.carbonlandtrust.com",
+  },
+  {
+    id: "nautilus",
+    label: "Nautilus wallet",
+    url: "https://github.com/nautls/nautilus-wallet",
+  },
+  {
+    id: "evernode",
+    label: "Evenode",
```

Comment:
Typo, missing the R.

```suggestion
    label: "Evernode",
```